Twig/NewsTwigExtension.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of TheCadien/SuluNewsBundle.
+ *
+ * (c) Oliver Kossin
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace TheCadien\Bundle\SuluNewsBundle\Twig;
+
+use Doctrine\Common\Cache\Cache;
+use TheCadien\Bundle\SuluNewsBundle\Repository\NewsRepository;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+/**
+ * Extension to handle news in frontend.
+ */
+class NewsTwigExtension extends AbstractExtension
+{
+    /**
+     * @var NewsRepository
+     */
+    private $newsRepository;
+
+    /**
+     * @var Cache
+     */
+    private $cache;
+
+    public function __construct(Cache $cache, NewsRepository $newsRepository)
+    {
+        $this->cache = $cache;
+        $this->newsRepository = $newsRepository;
+    }
```

Comment:
According to composer.json PHP 8.0 is the minimum required version. You could simplify this to:

```suggestion
    public function __construct(
        private readonly Cache $cache,
        private readonly NewsRepository $newsRepository
    ) {}
```